src/index.html

```diff
@@ -7,8 +7,53 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img
+        src="./images/logo.png"
+        alt="logo_MOYO"
+        class="header__images"
+        >
```

Comment:
<img width="1008" alt="image" src="https://user-images.githubusercontent.com/102293862/235749822-4af0537f-1080-44ec-94a8-08137ca7af2c.png">

```suggestion
        <img
          src="./images/logo.png"
          alt="logo_MOYO"
          class="header__images"
        >
```
Please, fix the code style everywhere and follow the checklist carefully.
https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md

Reply:
Done, fix the style code.